browser/src/slideshow/RenderContext.ts

```diff
@@ -75,6 +82,39 @@ class RenderContextGl extends RenderContext {
 		return texture;
 	}
 
+	public createEmptySlide(
+		width: number,
+		height: number,
+	): WebGLTexture | ImageBitmap {
+		const gl = this.getGl();
+		const texture = gl.createTexture();
+		if (!texture) {
+			throw new Error('Failed to create texture');
+		}
+		gl.bindTexture(gl.TEXTURE_2D, texture);
+
+		const blackPixels = new Uint8Array(width * height * 4).fill(0);
```

Comment:
Note to my self: Try  new Uint8Array([0, 0, 0, 255]); instade of height width ration. Will be faster.